src/test/borsh_structs.rs

```diff
@@ -31,15 +33,15 @@ pub struct TestStruct {
     skipped_field: Option<u32>,
 }
 
-#[derive(BorshSerialize, BorshDeserialize, Clone, Copy, Debug)]
-#[cfg_attr(test, derive(BorshSchema))]
+#[derive(BorshSchema, BorshSerialize, BorshDeserialize, Clone, Copy, Debug)]
+#[cfg_attr(test, derive(BorshSchemaTS))]
 pub struct OtherState {
     #[alias(u64)]
```

Comment:
`alias` and `schema_skip` attribute not available here if `BorshSchema` used instead of `BorshSchemaTS`